src/features/world/ui/chests/BudBox.tsx

```diff
@@ -188,16 +190,19 @@ export const BudBox: React.FC<Props> = ({ onClose, setIsLoading }) => {
                 <Label
                   icon={SUNNYSIDE.icons.stopwatch}
                   type="info"
-                  className="absolute -top-2 -right-2"
+                  className="absolute -top-3 -right-2"
                 >
-                  {`Today - ${secondsToString(secondsLeftToday, {
+                  {`${t("budBox.today")} - ${secondsToString(secondsLeftToday, {
```

Comment:
I suggest interpolation this entire string, as the order for some languages are different

Reply:
Nice catch, fixed.